### PR TITLE
feat:  Lat/Long fixes cheatsheet

### PR DESCRIPTION
**Pull request by Hypeism**
feat: Lat/Long fix conversion chart 

.latlongfix or .llfix
## Test Results

image

## Discord Username
HowNowBrownCrow#2591

### CODE REVIEW

**Comment by Hypeism** [comment]
SVG ocean_fixes
PNG oceanicfixes

**Comment by tracernz** [comment]
This missed the half-degree NAT waypoints. It should probably also cover what to do if there is no shorthand waypoint in the database for the one you want...

**Comment by Hypeism** [comment]
https://cdn.discordapp.com/attachments/897491699167793182/967661284508192778/unknown.png

PNG oceanicfixes
SVG fbw_oceanicfixes
SVG layer structure: 
image

poster made in accordance with design principles here: https://github.com/flybywiresim/branding/blob/master/branding-guidelines/FBW-Brand-Identity-Handbook.pdf

File: src/commands/index.ts [hunk 1]
```diff
@@ -223,9 +224,12 @@ const commands: CommandDefinition[] = [
     abbreviations,
     airac,
     mico,
+    translate,
```
Comment (benw202) [inline src/commands/index.ts:227]:
```suggestion
```

Repeated line? (Already in line 229)

File: src/commands/index.ts [hunk 2]
```diff
@@ -223,9 +224,12 @@ const commands: CommandDefinition[] = [
...
     salty,
     translate,
     github,
+    latlongfix
+
```
Comment (benw202) [inline src/commands/index.ts:232]:
```suggestion
```

Empty line added?

**Review by benw202 — CHANGES_REQUESTED** [review]

File: src/commands/general/latlongfix.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { CommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+import { makeEmbed } from '../../lib/embed';
+
```
Comment (benw202) [inline src/commands/general/latlongfix.ts:4]:
```suggestion

const LATLONG_IMAGE_URL = 'https://cdn.discordapp.com/attachments/817517626045104149/967659373054816286/unknown.png';

```

File: src/commands/general/latlongfix.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+export const latlongfix: CommandDefinition = {
+    name: ['latlong', 'llfix'],
+    description: 'Provides a cheat sheet for conversion between Latitude and longitude coordinates between short and long format',
+    category: CommandCategory.GENERAL,
+    executor: async (msg) => {
+        const latlongfixEmbed = makeEmbed({
+            title: 'FlyByWire | Latitude & Longitude Fix Conversion Chart',
+            description: 'If the point you\'re looking for isn\'t part of the database, check out our guide on [adding and storing custom waypoints.](https://docs.flybywiresim.com/pilots-corner/advanced-guides/data-management/?h=data+man#format-latitude-longitude-ll)',
+            image: { url: 'https://cdn.discordapp.com/attachments/817517626045104149/967659373054816286/unknown.png' },
```
Comment (benw202) [inline src/commands/general/latlongfix.ts:13]:
```suggestion
            image: { url: LATLONG_IMAGE_URL },
```

**Review by benw202 — CHANGES_REQUESTED** [review]

**Review by benw202 — APPROVED** [review]

**Pull request merged**